tests/cases/fourslash/jsDocGenerics1.ts

```diff
@@ -0,0 +1,10 @@
+///<reference path="fourslash.ts" />
+
+// @allowNonTsExtensions: true
+// @Filename: Foo.js
+//// /** @type {Array<number>} */

```

Comment:
Maybe for completeness something a little more complex, e.g. nested type args or an object property, e.g. `{PromiseLike<Array<number>>}` or `{{a: string, b: Array<number>}}`. Maybe also an optional param, e.g. `@param {Array<number>?} arg1`